mesonbuild/compilers/fortran.py

```diff
@@ -14,6 +14,7 @@
 from typing import List
 import subprocess, os
 
+from typing import List
```

Comment:
__[Flake8]__ [F811] redefinition of unused 'List' from line 14

<sub>posted by Sider</sub>